index.html

```diff
@@ -51,5 +52,183 @@ <h3>LET’S CONNECT</h3>
+          sign-ups required.
+        </p>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">See Project</button>
+      </div>
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h2>Tonic</h2>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">See Project</button>
+      </div>
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h2>Tonic</h2>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+    </section>
+    <section class="about-myself">
+      <h2>About Myself</h2>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hesitate to
+        contact me.
+      </p>
+      <h3>LET’S CONNECT</h3>
+      <ul class="social-media">
+        <li><img src="/images/twitter-icon.svg" alt="twitter" /></li>
+        <li class="social-media-spacer">
+          <img src="/images/Linkedin icon.svg" alt="Linkedin" />
+        </li>
+        <li class="social-media-spacer">
+          <img src="/images/m.svg" alt="m" />
+        </li>
+        <li class="social-media-spacer">
+          <img src="/images/github.svg" alt="github" />
+        </li>
+        <li class="social-media-spacer">
+          <img src="/images/chill.svg" alt="chill" />
+        </li>
+      </ul>
+      <button class="margin-spacer">Get my resume</button>
```

Comment:
- [ ] You have done a great job so far but I think you still have a bit of requirements to meet before moving to the next milestone. :+1: :+1: 
- This was already suggested by the previous reviewer, As you can see in the video attached below, Your button does not have the `hover or active ` state styling, You need to fix this as your project is meant to follow all the given designs and requirements. :+1: :100: 

You can achieve this by adding this block of codes to your CSS file `button:active {
  background-color: #2230D2;
  color: #fff;
}` and `button:hover {
  background-color: #6070FF;
  color: #fff;
}`

- Video Reference:

https://user-images.githubusercontent.com/107632780/235326729-a20318b3-4e56-4c80-9efd-6eb89cb9efbe.mp4